### DISCUSSION TITLE
Using Prima Studio in a Front End App

### DISCUSSION THREAD

**Discussion by VinayaSathyanarayana** [prisma/prisma#5506] [Q&A]
How can I embed Prima Studio inside my front end app?
I would like the ability to restrict 
a. Rows and Columns Shown based on User Role permissions
b. When a User Inserts some data, I would like some hidden columns populated based on User Data

**Comment by janpio** [comment]
That is not a use case currently supported by Prisma Studio.

If you have a concrete request, please open an issue over at https://github.com/prisma/studio/. Thanks.
